src/intelfocl/AOCLUtils/opencl.h

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2013-2018 Altera Corporation, San Jose, California, USA. All rights reserved.
```

Comment:
I think a better place for source code files in this directory should be the `3rdparty` directory.

Reply:
Thank you for the comment. Sure, we can move it to 3rdparty directory as suggested.